devices/apa102/apa102_test.go

```diff
@@ -636,15 +636,15 @@ func TestInit(t *testing.T) {
 
 //
 
-func BenchmarkWriteWhite(b *testing.B) {
-	b.ReportAllocs()
-	pixels := [150 * 3]byte{}
-	for i := range pixels {
-		pixels[i] = 0xFF
+type fn func(int) byte
+
+func benchmarkWrite(b *testing.B, o Opts, length int, f fn) {
```

Comment:
I'd prefer if length was in number of pixels, not in the length of the buffer. This is confusing IMHO.

Reply:
Done.